include/net/aws_fota.rst

```diff
@@ -64,6 +63,33 @@ The implementation uses a job document similar to the following (where *bucket_n
 
 The current implementation uses information from the ``host`` and ``path`` fields only.
 
+For upgrading MCUBoot you'll have to do a similar process as the above except you'll have to turn on `CONFIG_SECURE_BOOT` and `CONFIG_MCUBOOT_BUILD_S1_VARIANT` for your application this will result in two files being generated when building your application which ends in *s0/s1_image_update.bin* in the build folder of your application. The URL of these files are added to the path starting with the *s0* file followed by the *s1* file where the path is separated by a space.
+
+.. note::
+  It is important that you place the path of *s0* before *s1* in the job document.
+
+
+.. parsed-literal::
+   :class: highlight
+
+   {
+     "operation": "bootloader_fw_update",
+     "fwversion": "v1.0.2",
+     "size": 241124,
+     "location": {
+       "protocol": "http:",
+       "host": "*bucket_name*.amazonaws.com",
+       "path": "*s0*.bin *s1*.bin"
+      }
+   }
+
+
+.. warning::
+  Not available in NCS 1.2
+
+
+If you are using nRF Cloud in combination with the AWS FOTA library in an application you can use the *dfu_mcuboot.zip* file generated by turning on `CONFIG_SECURE_BOOT` and chose bootloader through the same interface you are using for application and modem upgrades. Here you can upload the zip file from your project and nRF Cloud will handle the rest.
```

Comment:
This seems to be the most important part, but I think we need more information (we don't have information about updates through nRF Cloud at all). Let's discuss.

Reply:
Yeah, so this should probably be put into it's own chapter describing the FOTA process for nRF cloud I guess that would go for the above comment also.

Reply:
Yes, I think we need documentation about how to do FOTA with nRF Cloud. Not quite sure where this should go (maybe into https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160.html#fota-upgrades ), but this should cover the updates here.

Reply:
Tried to add some instructions to this section. It's not really too well written. See the issue with updated images on what I tried to describe.